rllib/offline/dataset_reader.py

```diff
@@ -251,12 +252,34 @@ def next(self) -> SampleBatchType:
             d = next(self._iter).as_pydict()
             # Columns like obs are compressed when written by DatasetWriter.
             d = from_json_data(d, self._ioctx.worker)
-            d = postprocess_actions(d, self._ioctx)
+            d = postprocess_actions(d, self._ioctx.worker)
             count += d.count
+            d = self._preprocess_if_needed(d)
```

Comment:
should we re-arrange these to be:
```
...
count += d.count
d = self._preprocess_if_needed(d)
d = postprocess_actions(d, self._ioctx.worker)
d = self._postprocess_if_needed(d)
ret.append(d)
```
so it looks like preprocessing happens before postprocessing??

Reply:
good idea.